manifests/init.pp

```diff
@@ -4,13 +4,15 @@
 #
 # === Parameters:
 #
-# $enable_yum::         Enable rpm content plugin, including syncing of yum content
+# $enable_yum::           Enable rpm content plugin, including syncing of yum content
 #
-# $enable_file::        Enable generic file content management
+# $enable_file::          Enable generic file content management
 #
-# $enable_docker::      Enable docker content plugin
+# $enable_docker::        Enable docker content plugin
 #
-# $enable_deb::         Enable debian content plugin
+# $enable_deb::           Enable debian content plugin
+#
+# $enable_katello_agent:: Enable use of katello-agent for remote yum actions
```

Comment:
I wonder if it makes sense to name it
```suggestion
# $enable_agent:: Enable use of katello-agent for remote yum actions
```

Reply:
I know it might seem redundant, since the module has katello already in and it the parameter translated in the installer would also start with katello, but I feel like katello-agent is so widely known as a term that we shouldn't shorten it to just agent. I would want no ambiguity here.

Reply:
This reasoning is sound to me. Particularly when an user can encounter these parameters in a setting (foreman-installer) which is pretty removed from these modules, I see a clear benefit to being a little verbose with parameter names.